buildsystem/modules/FindPython.cmake

```diff
@@ -43,6 +43,29 @@ function(py_get_config_var VAR RESULTVAR)
 	set("${RESULTVAR}" "${RESULT}" PARENT_SCOPE)
 endfunction()
 
+function(py_get_lib_name RESULTVAR)
+	# uses py_exec to compute Python's C/C++ library name, just like python-config does.
+	py_get_config_var(VERSION PYTHON_VERSION)
+	if(NOT "${PYTHON_VERSION}" VERSION_LESS "3.2")
+		set(USE_ABIFLAGS "use")
```

Comment:
Why should it ever be called? we explicitly search for `py>=3.4` which will already fail if not found.